src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds/DependencyResolution/RefundsRegistry.cs

```diff
@@ -16,8 +16,8 @@ public RefundsRegistry(Type taskType, ContextWrapper contextWrapper)
             Scan(
                 scan =>
                 {
-                    scan.TheCallingAssembly();
-                    
+                    scan.AssemblyContainingType<RefundsRegistry>();
```

Comment:
those two lines should be doing exactly the same thing. but I'm happy if it fixes any weird DC IOC issues.